props/saber_fett263_buttons.h

```diff
@@ -4311,7 +4317,11 @@ SaberFett263Buttons() : PropBase() {}
 
   RefPtr<BufferedWavPlayer> wav_player;
 
+#ifndef FETT263_QUICK_SELECT_ON_BOOT
   float current_menu_angle_ = 0.0;
+#else
+  float current_menu_angle_ = fusor.angle2();
```

Comment:
I don't think this will work, as this assignment will be executed before the fusor has had a chance to actually do anything.

Reply:
I have updated and moved the current_menu_angle_ into Loop with a delay for boot to allow fusor startup.